### PR TITLE
Re #2671: fixes difference between top and main LCD TX battery bars, …

### PR DESCRIPTION
**Pull request by projectkk2glider**
…code made more readable with macros

Re #2671

### CODE REVIEW

File: radio/src/gui/9X/lcd.cpp [hunk 1]
```diff
@@ -587,7 +587,7 @@ void drawFilledRect(coord_t x, scoord_t y, coord_t w, coord_t h, uint8_t pat, Lc
 void lcdDrawTelemetryTopBar()
 {
   putsModelName(0, 0, g_model.header.name, g_eeGeneral.currModel, 0);
-  uint8_t att = (g_vbat100mV < g_eeGeneral.vBatWarn ? BLINK : 0);
+  uint8_t att = (IS_TXBATT_WARNING() ? BLINK : 0);
```
Comment (projectkk2glider) [inline radio/src/gui/9X/lcd.cpp]: There was a small inconsistence: sometimes  `g_vbat100mV < g_eeGeneral.vBatWarn` was used for a warning treshold and sometimes `g_vbat100mV <= g_eeGeneral.vBatWarn` was used. Now they are both the same.

File: radio/src/gui/9X/view_main.cpp [hunk 2]
```diff
@@ -223,15 +223,15 @@ void displayBattVoltage()
...
     putsValueWithUnit(6*FW-1, 2*FH, getTemperature(), UNIT_TEMPERATURE, BLINK|INVERS|DBLSIZE);
   }
-  else if (g_vbat100mV > g_eeGeneral.vBatWarn && g_eeGeneral.mAhWarn && (g_eeGeneral.mAhUsed + Current_used * (488 + g_eeGeneral.currentCalib)/8192/36) / 500 >= g_eeGeneral.mAhWarn) {
+  else if (g_eeGeneral.mAhWarn && (g_eeGeneral.mAhUsed + Current_used * (488 + g_eeGeneral.currentCalib)/8192/36) / 500 >= g_eeGeneral.mAhWarn) {
     putsValueWithUnit(7*FW-1, 2*FH, (g_eeGeneral.mAhUsed + Current_used*(488 + g_eeGeneral.currentCalib)/8192/36)/10, UNIT_MAH, BLINK|INVERS|DBLSIZE);
   }
-  else {
-    displayBattVoltage();
-  }
 }
```
Comment (projectkk2glider) [inline radio/src/gui/9X/view_main.cpp]: Code rearranged, but should give the same result with less instructions while being more readable.

File: radio/src/targets/taranis/top_lcd_driver.cpp [hunk 3]
```diff
@@ -347,13 +347,13 @@ void setTopBatteryState(int state, uint8_t blinking)
...
     if (state > 2)
+      Ht1621Data1[9] |= 0x80;
+    if (state > 4)
       Ht1621Data1[5] |= 0x80;
-    if (state > 3)
+    if (state > 6)
       Ht1621Data1[10] |= 0x80;
-    if (state > 4)
+    if (state > 8)
       Ht1621Data1[4] |= 0x80;
```
Comment (projectkk2glider) [inline radio/src/targets/taranis/top_lcd_driver.cpp]: With this numbers the bars display is now equal on both LCDs.

**Comment by bsongis** [comment]
Rebased and pushed, thanks!

**Pull request merged**